addons/l10n_ar_withholding/demo/product_product_demo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product.product_product_2" model="product.product">
+            <field name="l10n_ar_supplier_withholding_taxes_ids" search="[('l10n_ar_withholding', '=', 'supplier'), ('name', 'like', '%CABA%')]"/>
+    </record>
```

Comment:
try to avoid matching on name (translations, users changing stuff, ...)

Reply:
Would still be good to check if we could rather use the Python demo generation mechanism.  This XML Python is not ideal.

Reply:
This file is droped